Sming/Makefile-windows.mk

```diff
@@ -9,7 +9,7 @@ SDK_BASE	?= $(ESP_HOME)/ESP8266_SDK
 SDK_TOOLS	 ?= $(ESP_HOME)/utils
 
 # Other tools mappings
-ESPTOOL		 ?= $(SDK_TOOLS)/esptool.exe
+ESPTOOL		 ?= $(SDK_TOOLS)/ESP8266/esptool.exe
```

Comment:
> you can define it directly in your environment without the need to change it in the Sming source code
Yes, but current version didn't work under any UDK what is wrong. SO some solution was needed anyway, I think.

Lat's contnue discuss this issue in related topic: https://github.com/SmingHub/Sming/issues/938